src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,91 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int RESIZE_FACTOR = 2;
+    private Node<K, V> []table;
+    private int size;
+    private int threshold;
+    private int capacity;
+
+    public MyHashMap() {
+        this.table = new Node[DEFAULT_CAPACITY];
+        this.capacity = DEFAULT_CAPACITY;
+        this.threshold = (int) (DEFAULT_CAPACITY * LOAD_FACTOR);
```

Comment:
```suggestion
        table = new Node[DEFAULT_CAPACITY];
```
we don't need `this`. [Here's](https://stackoverflow.com/a/2411283) where it's needed